src/client/app/redux/selectors/barChartSelectors.ts

```diff
@@ -12,15 +12,13 @@ import getGraphColor from '../../utils/getGraphColor';
 import { createAppSelector } from './selectors';
 import { selectAreaScalingFromEntity, selectNameFromEntity } from './entitySelectors';
 import { selectPlotlyMeterDeps, selectPlotlyGroupDeps } from './plotlyDataSelectors';
```

Comment:
I'm wondering if this change about selectSelectedLanguage is showing since that change not yet merged into this PR or something else? Either way, it is a needed change so figuring out would be good.

Reply:
I'm noting that this is resolved but it did change the git record to show it as a recent change. The work was originally done in PR #1547.